bzira

```diff
@@ -317,9 +317,9 @@ jira_workspace="${JIRA_WORKSPACE:-$default_jira_workspace}"
 jira_project="${JIRA_PROJECT:-$default_jira_project}"
 jira_board="${JIRA_BOARD:-$default_jira_board}"
 # shellcheck disable=SC2153
```

Comment:
neat!
you can probably get rid of those two `# shellcheck disable=SC2153` now

Reply:
done